### ISSUE TITLE
Flags to disable Wallet and Rewards

### ISSUE THREAD

**Issue by hyperion4** [brave/brave-browser#41181]
### Platforms

all

### Description

There are policies to disable Rewards, Wallet, Leo AI and VPN.
Policies are nice to have, but in many cases people just don't want to use policies.
Policies turn your Brave installation to managed and DoH has to be also managed with policies.
Also they do not work in Android.
Brave has flags to disable Leo AI and VPN and they can be disabled as features.
So you can disable them from commandline.
brave.exe  --disable-features=AIChat,BraveVPN
When you launch brave with this commandline you have Leo AI and VPN disabled like they are disabled with their policies.
But I can't find the flags and the feature names of Wallet and Rewards.
Can we disable Wallet and Rewards as features with chromium's --disable-features commandline option like we can with Leo and VPN?
If we can't, please consider adding the needed flags for them too like we can disable Leo and VPN.
Policies are not the best solution for many people, they need admin rights and they lock DoH.
Thank you in advance for any reply.

**Comment by thericle** [comment]
Although I was able to find the flag for disabling Leo AI (thank you for that), I couldn't find one for BraveVPN. I also found a flag named "Enable Brave Wallet" (#native-brave-wallet) but disabling it didn't do anything, I think.

**Comment by hyperion4** [comment]
> Although I was able to find the flag for disabling Leo AI (thank you for that), I couldn't find one for BraveVPN. I also found a flag named "Enable Brave Wallet" (#native-brave-wallet) but disabling it didn't do anything, I think.

Yes, I did a check and they removed the flag that disables the VPN.
Unbelievable lol.
But it still works from the commandline, the following will still disable the VPN like the BraveVPNDisabled policy.
brave.exe --disable-features=BraveVPN
It is clear to me because they removed the flag that disables the VPN, that they don't want people to disable easily what they can disable with policies.
What they fail to undestand is what policies are lol. Policies are for FORCING an option to the browser.
In all browsers there is no policy that doesn't have a same option in the browser (as a flag, inside about:config).
Even Edge, a browser from a big corporation, has options in the browser apart from policies to disable everything they keep and keep adding on it.

**Comment by bsclifton** [comment]
Thanks for the request - closing as duplicate of https://github.com/brave/brave-browser/issues/43030
